spanner/api/Spanner.Samples.Tests/DeleteUsingDmlReturningAsyncPostgreTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Data;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class DeleteUsingDmlReturningAsyncPostgreTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public DeleteUsingDmlReturningAsyncPostgreTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public async Task TestDeleteUsingDmlReturningAsyncPostgre()
+    {
+        InsertDataAsync();
+        var sample = new DeleteUsingDmlReturningAsyncPostgreSample();
```

Comment:
We can probably initialize the sample in the constructor itself. However, it is fine if we want to keep it as is.

Reply:
yes, I kept it like this to keep test class format consistent with existing sample tests.